integrated_channels/sap_success_factors/admin/__init__.py

```diff
@@ -0,0 +1,42 @@
+"""
+Django admin integration for sap_success_factors app.
```

Comment:
If we can provide some more context here describing the actual purpose for this app, that would be great -- please include a link to pertinent documentation @ SAP, such as the OCN integration guide.